docs/server/source/cloud-deployment-templates/ca-installation.rst

```diff
@@ -2,81 +2,30 @@ Set Up a Self-Signed Certificate Authority
 ==========================================
 
 This page enumerates the steps *we* use to set up a self-signed certificate authority (CA).
-Your organization might do it differently.
+This is something that only needs to be done once per cluster,
+by the organization managing the cluster, i.e. the CA is for the whole cluster.
+We use Easy-RSA.
```

Comment:
That is one more reason why I had the directory names as `federation-ca` and `client-cert` and `server-cert`.